rules/integrations/kubernetes/execution_user_exec_to_pod.toml

```
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2022/05/17"
+maturity = "production"
+updated_date = "2022/05/17"
+integration = "kubernetes"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects a user attempt to exec into a pod or container. Execing into a pod allows a user to execute any process in the container which is not already running. A user may execute the bash process to gain an interactive shell which will allow access to any data the pod has permissions to, including secrets. 
```

```suggestion
This rule detects a user attempt establish a shell session into a pod or container using the 'exec' command. Using the 'exec' command in a pod allows a user to establish temporary shell session and execute any process/commands in the container at the pod level. An adversary may call bash to gain a persistent interactive shell which will allow access to any data the pod has permissions to, including secrets. 
```

Did a bit of reading. Does this make sense for potential changes? From what I understand it is similar to docker'a `exec -it` which spawns an interactive shell or allows commands be run interactively within the container.

---

Yes that makes sense and sounds much clearer, thank you Terrance!